docs/visual-basic/programming-guide/language-features/data-types/value-types-and-reference-types.md

```diff
@@ -12,7 +12,7 @@ helpviewer_keywords:
 ms.assetid: fc82ce15-5a40-4c5c-a1e1-a556830e7391
 ---
 # Value Types and Reference Types
-In Visual Basic, data types are implemented based on their classification. The Visual Basic data types can be classified according to whether a variable of a particular type stores its own data or a pointer to the data. If it stores its own data it is a *value type*; if it holds a pointer to data elsewhere in memory it is a *reference type*.  
+There are two kinds of types in Visual Basic: reference types and value types. Variables of reference types store references to their data (objects), while variables of value types directly contain their data. With reference types, two variables can reference the same object; therefore, operations on one variable can affect the object referenced by the other variable. With value types, each variable has its own copy of the data, and it is not possible for operations on one variable to affect the other (except in the case of [ByRef modifier on parameters](https://docs.microsoft.com/en-us/dotnet/visual-basic/language-reference/modifiers/byref)).
```

Comment:
```suggestion
There are two kinds of types in Visual Basic: reference types and value types. Variables of reference types store references to their data (objects), while variables of value types directly contain their data. With reference types, two variables can reference the same object; therefore, operations on one variable can affect the object referenced by the other variable. With value types, each variable has its own copy of the data, and it is not possible for operations on one variable to affect the other (except in the case of [ByRef modifier on parameters](../../../language-reference/modifiers/byref)).
```

Two suggestions here: One to remove the `en-us` locale provider. The second to change to a relative link, which provides build validation for the link.

Reply:
One additional suggestion: precede ByRef with "the".